docs/development/extensions-core/s3.md

```diff
@@ -32,7 +32,7 @@ To use this Apache Druid extension, [include](../../development/extensions.md#lo
 
 ### Reading data from S3
 
-Use a native batch [Parallel task](../../ingestion/native-batch.md) with an [S3 input source](../../ingestion/native-batch-input-sources.html#s3-input-source) to read objects directly from S3.
```

Comment:
Should this change from plural to singular? (sources to source) Also in line 82

Reply:
Yes, the id differs from the filename

![image](https://github.com/apache/druid/assets/53799971/4a2c91b3-2343-45fd-9853-466e7e57d0e2)